src/main/java/md/maib/controller/CustomerController.java

```diff
@@ -0,0 +1,23 @@
+package md.maib.controller;
+
+import java.util.List;
+
+import md.maib.entity.Customer;
+import md.maib.repository.CustomerRepository;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
```

Comment:
How is it common with **DB Setup** ?

Reply:
i checked if the testcontainer works

Reply:
I don't care. It is not a part of DB setup, so it have to be excluded

Reply:
you can check db layer without rest part